From: Ben Hutchings <ben@decadent.org.uk>
Subject: media: Avoid ABI change in 3.10.13
Forwarded: not-needed

genksyms tends to change symbol versions when a new #include is added.
I didn't see any changes on amd64, but just to be careful let's hide
this.

--- a/include/media/v4l2-ctrls.h
+++ b/include/media/v4l2-ctrls.h
@@ -22,7 +22,9 @@
 #define _V4L2_CTRLS_H
 
 #include <linux/list.h>
+#ifndef __GENKSYMS__
 #include <linux/mutex.h>
+#endif
 #include <linux/videodev2.h>
 
 /* forward references */
